Documentation/backends.md

```diff
@@ -115,4 +115,5 @@ IPIP kind of tunnels is the simplest one. It has the lowest overhead, but can in
 
 Type:
 * `Type` (string): `ipip`
+* `MTU` (number): The mtu of ipip device. Defaults to 1480.
```

Comment:
Can you write a little more here to help guide users. Why is this this value? Why and when would a user want to change it? And what value would they change it to?

Reply:
Fixed